src/cogent3/app/sqlite_data_store.py

```diff
@@ -271,6 +272,9 @@ def drop_not_completed(self) -> None:
         self._not_completed = []
 
     def write(self, *, unique_id: str, data: str) -> DataMember:
+        if unique_id.startswith(_RESULT_TABLE):
+            unique_id = Path(unique_id).name
+
```

Comment:
Interestingly, today I was thinking about this.
I think wo should not need to add table_name in calling all write functions.
We both was thinking on this topic at the same time.
If we can get rid of suffix, it will be great.

Reply:
Not having the table name as part of the `unique_id` is appealing. All it should require is that our read method be modified to discriminate between logs and results. I will look at this.